src/Scene.js

```diff
@@ -326,9 +328,12 @@ Scene.prototype.lookTo = function(params, opts, done) {
 
   var controlsEnabled = this._viewer.controls().enabled();
 
-  this._viewer.controls().disable();
+  if(!controlsInterrupt) {
+    this._viewer.controls().disable();
+  }
+
   this.startMovement(movement, function() {
-    if(controlsEnabled) {
+    if(controlsEnabled && !this._viewer.controls().enabled()) {
```

Comment:
Add a space between the `if` and the `(`.

I don't think you need to check that the controls are enabled, since enabling again should be a no-op?

Reply:
Without the check for current status of `controls` I encountered an error during testing. When trying to quickly trigger controls to change the view right after the transition finishes, I was getting `Uncaught RangeError: Maximum call stack size exceeded`. This fixed it.

Reply:
That suggests a bug elsewhere, but that's fine - let's keep the check for now, and I can investigate later.